en_US/changes/changes-ee-5.4.md

```diff
@@ -0,0 +1,95 @@
+# Releases
+
+## e5.4.0
+
+### Enhancements
```

Comment:
```suggestion
### Enhancements

	[#12201](https://github.com/emqx/emqx/pull/11994) Support hot update of TCP/SSL/WS/WSS MQTT listeners configuration.
	
	 This allows changing most of the configuration parameters without restarting the listener and disconnecting the clients.
	
	In case of TCP/SSL listeners, changes to the following parameters still require listener restart and clients reconnect:
	 * `bind`
	 * `tcp_options.backlog`
	
	In case of WS/WSS listeners, any parameter can be freely changed without losing the connected clients. However, changing transport related parameters will cause listening socket to be re-opened, namely:
	 * `bind`
	 * `tcp_options.*`
	 * `ssl_options.*`
```